src/main/java/uk/gov/pay/api/model/RequestError.java

```diff
@@ -127,26 +129,36 @@ public static RequestError aRequestError(Code code, Object... parameters) {
     }
 
     public static RequestError aRequestError(String fieldName, Code code, Object... parameters) {
-        return new RequestError(fieldName, code, parameters);
+        return new RequestError(null, fieldName, code, parameters);
+    }
+
+    public static RequestError aRequestErrorHeader(String header, Code code, Object... parameters) {
```

Comment:
would `aHeaderRequestError` be a better name - as it's not a header that's being returned

Reply:
👍